zvariant/src/object_path.rs

```diff
@@ -207,9 +207,9 @@ impl<'a> PartialEq<&str> for ObjectPath<'a> {
     }
 }
 
-impl<'a> Debug for ObjectPath<'a> {
+impl Debug for ObjectPath<'_> {
```

Comment:
nit: this line change is irrelevant to the containing commit. It's a tiny change so I leave it to you to decide whether to split or not. :)

Reply:
I don't know how to explain this change so I left it as is for now.

Reply:
"Switch to anonymous lifetimes for an impl" is fine.

Reply:
I decided to revert this change to maintain consistency with surrounding code.

Reply:
ok